examples/bq-remote-function/README.md

```diff
@@ -0,0 +1,18 @@
+# BigQuery Remote Function Java Samples
+
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+[shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
+[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/K4singh/professional-services&page=editor&open_in_editor=examples/bq-remote-function/string_format
+
+[Bigquery remote functions](https://cloud.google.com/bigquery/docs/reference/standard-sql/remote-functions) allows you to use your custom services or libraries written in any language other than SQL and javascript which are not present as bigquery user defined functions. 
+This repository has sample code for setting up bigquery remote function. it contains simple string format code, the custom function is deployed as cloud run gen 2 which can be invoked using api call.
+
+###
+[<img src="https://storage.googleapis.com/cloudrun/button.svg" alt="Run on Google Cloud" height="30">][run_button_helloworld] 
+###
+
+For more Cloud Run samples beyond Java, see the main list in the [Cloud Run Samples repository](https://github.com/GoogleCloudPlatform/cloud-run-samples).
+
```

Comment:
Please make sure to include manual setup steps via gcloud command, as well include example BigQuery code on how a user can test out the format function.

Reply:
added!